lib/temporal/metadata.rb

```diff
@@ -59,6 +36,23 @@ def workflow_task_metadata_from(task, namespace)
           workflow_name: task.workflow_type.name
         )
       end
+
+      # event: Temporal::Workflow::History::Event
+      # event.attributes: WorkflowExecutionStartedEventAttributes
+      # task_metadata: Temporal::Metadata::WorkflowTask
```

Comment:
Can you please use the YARD format here? `https://github.com/coinbase/temporal-ruby/blob/master/lib/temporal/client.rb` has examples